test/blackbox-tests/test-cases/coq/compose-boot-nostdlib.t/run.t

```diff
@@ -0,0 +1,36 @@
+Testing composition of theories accross a dune workspace with a boot library and
+importing stdlib enabled or disabled.
+
+Composing a library A depending on Coq but having `(stdlib no)`:
```

Comment:
Would it be okay to write "Composing library A..." instead of "a library A"? If so, please also change the following one for composing library B.